opennms-webapp/src/main/web-filtered/WEB-INF/version.properties

```diff
@@ -26,6 +26,5 @@
 #     http://www.opennms.com/
 ###############################################################################
 
-version.display=${display.version}
-opennms.product=meridian
-release.version=2024
\ No newline at end of file
+version.display=2024.1.3
```

Comment:
We should not hardcode the version here. The `${display.version}` is set from the pom.xml using `${install.version}` and follows automatically the correct version number. Hardcoding the version number would require dealing with it manually.